crowbar_framework/config/routes.rb

```diff
@@ -240,4 +236,48 @@
   # TODO(must): Get rid of this wildcard route
   match "/:controller/:action/*(:.format)",
     via: [:get, :post, :put, :patch, :delete]
+
+  namespace :api,
```

Comment:
@kirushik OK thanks, that makes perfect sense and aligns with what I was thinking regarding controller reuse. @MaximilianMeister in this context, how much commonality is there between the UI and API regarding usage patterns and controller code?